node/src/chain_specs/mod.rs

```diff
@@ -18,7 +18,7 @@
 #![allow(dead_code)]
 use cumulus_primitives_core::ParaId;
 use hex_literal::hex;
-use manta_primitives::{constants, AccountId, AuraId, Balance, Signature};
+use manta_primitives::{constants, types::{AccountId, AuraId, Balance, Signature}};
```

Comment:
You can add a `pub` to this line of code, like `pub use manta_primitives::{...}`, so you don't change these codes like `use manta_primitives::types::Header;`.

Anyway, it works for me.

Reply:
resolved as per discussion

> I thought about reexporting, but I personally like clarity over brevity: Knowing that Balance is a types:: and not a constants:: on first glance in the use declaration is worth the extra verbosity for my taste.